Assets/Mirror/Migration/Editor/Scripts.cs

```diff
@@ -132,12 +131,16 @@ public class Scripts : MonoBehaviour {
             StreamWriter sw;
 
             foreach (string file in filesToProcess) {
-                try {
+                try
+                {
+                    Encoding encoding = Utils.GetEncoding(file);
                     // Open and load it into the script buffer.
-                    using (sr = new StreamReader(file)) {
+                    using (sr = new StreamReader(file, encoding)) {
                         scriptBuffer = sr.ReadToEnd();
                     }
 
+                    if (scriptBuffer.Contains("namespace Mirror")) continue;
```

Comment:
Don't modify files in Mirror namespace, they will be 100% correct and converter only can break them (as it currently does)